Salary App/Features/ValidationManager/ValidationManager.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ValidationManager.swift
+//  Salary App
+//
+//  Created by Tobi Adegoroye on 12/07/2023.
+//
+
+import Foundation
+
+enum FormError: Error {
+    case containsEmptyString
+    case formContainsLetters
+    case invalidSalary
+    case isValidPenContribution
+}
+
+extension FormError: LocalizedError {
+    var errorLocalDescription: String {
```

Comment:
You shouldnt be creating your own computed property. When working with localizederror you can override the computed properties they offer check out this tutorial https://figure.ink/2021/07/18/practical-localized-error.html